### PR TITLE
Winnyschuster correct type of ts in user queries

### PR DESCRIPTION
**Pull request by winnyschuster**
For postgresql and ms sqlserver, type of ts in user definded queries is string instead of number. This PR will fix this

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -616,6 +616,12 @@ function _userQuery(msg, callback) {
                 client.execute(msg.message, (err, rows /* , fields */) => {
                     if (rows && rows.rows) rows = rows.rows;
                     returnClientToPool(client);
+                    //convert ts for postgresql and ms sqlserver
+                    if (!err && rows && rows[0].ts && typeof rows[0].ts === 'string') {
+                        for (let i = 0; i < rows.length; i++) {
+                            rows[i].ts = parseFloat(rows[i].ts);
```
Comment (Apollon77) [inline main.js:622]: Is the Really a float? Should be  better parseInt, or?

Comment (Apollon77) [reply main.js:622]: Additionally it seems we need another guard (https://github.com/ioBroker/ioBroker.sql/runs/5838765177?check_suite_focus=true#step:6:910)

... also check for rows[0] before accessing rows[0].ts

Comment (winnyschuster) [reply main.js:622]: upps, of course it schould be parseInt(), thank you! 
i was assuming when we have rows we also have at least rows[0] too, don't we?

what is a guard? i can see some errors but honestly don't know what to do with them. However, would appreciate if you could guide me thru with some hints so i can do better in future.

Comment (winnyschuster) [reply main.js]: i did what you suggested and tests have all passed now (22/22 checks ok). Outstanding changes to Readme.md will follow in a separate PR later/end of week

Comment (Apollon77) [reply main.js]: >  i was assuming when we have rows we also have at least rows[0] too, don't we?

Only if there is a response ... empty set also do not contain a [0] element

Guard in this case I mean a check of existenceof rows[0] before accessing a property of it

Comment (winnyschuster) [reply main.js]: ah.... ok, thanks for explanation

**Pull request merged**